overlays/minikube/kustomization.yaml

```diff
@@ -96,6 +96,24 @@ patchesJson6902:
       group: apps
       version: v1
     path: delete-resources.yaml
+  - target:
+      kind: Deployment
+      name: codeintel-db
+      group: apps
+      version: v1
+    path: delete-resources-2.yaml    
+  - target:
+      kind: Deployment
+      name: codeinsights-db
+      group: apps
+      version: v1
+    path: delete-resources.yaml  
+  - target:
+      kind: Deployment
+      name: minio
+      group: apps
+      version: v1
+    path: delete-resources.yaml      
```

Comment:
I believe this will remove code-intel from the `minikube` overlay? Just wanted to note that